iamclient/iamclient.go

```diff
@@ -640,6 +724,24 @@ func (client *Client) connectPublicNodeService() {
 	}
 }
 
+func (client *Client) subscribeUnitSubjectsChange() (
+	listener pb.IAMPublicIdentityService_SubscribeSubjectsChangedClient, err error,
+) {
+	client.Lock()
+	defer client.Unlock()
+
+	client.identService = pb.NewIAMPublicIdentityServiceClient(client.publicConnection)
+
+	listener, err = client.identService.SubscribeSubjectsChanged(context.Background(), &emptypb.Empty{})
+	if err != nil {
+		log.WithField("error", err).Error("Can't subscribe on unit subjects change event")
```

Comment:
Keep in go projects displaying err as `Can't subscribe on unit subjects change event: %v", err`